.github/workflows/maven.yml

```diff
@@ -0,0 +1,23 @@
+name: Java CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
```

Comment:
Is this the branch the PR is made to or from? It looks like you have a passing build on this PR, so I guess it's the branch you're merging to?

Reply:
I think its the branch I'm merging to yes.